src/visualizers/VisualizerInterface.ts

```diff
@@ -7,26 +7,25 @@ interface VisualizerConstructor {
      * @param seq SequenceInterface The initial sequence to visualize
      */
     new (seq: SequenceInterface): VisualizerInterface
-    visualizationName: string
 }
 
 export class VisualizerExportModule {
+    visualizer: VisualizerConstructor
     name: string
     description: string
-    visualizer: VisualizerConstructor
 
-    constructor(viz: VisualizerConstructor, description: string) {
-        this.name = viz.visualizationName
+    constructor(
+        viz: VisualizerConstructor,
+        name: string,
+        description: string
+    ) {
         this.visualizer = viz
+        this.name = name
         this.description = description
     }
```

Comment:
Right, here is where you should be able to set this.name from viz.protototype.name, right? (And similarly for description)